docs/adr/0009-cluster-resources.md

```diff
@@ -0,0 +1,95 @@
+# Declarative definitions for cluster scoped resources
+
+- Status: accepted
+- Deciders: 4n4nd, anishasthana, HumairAK, tumido
+- Date: 2021-01-18
+
+Technical Story: [issue-1](https://github.com/operate-first/odh-moc-support/issues/39), [issue-2](https://github.com/open-infrastructure-labs/ops-issues/issues/2), [pr-2](https://github.com/operate-first/apps/pull/104)
+
+## Context and Problem Statement
+
+A full Git-Ops driven approach we decided to take in the previous ADRs, makes us face additional challenges when deploying cluster scoped resources. Resources like `Namespace`, `Group` etc. should be also defined declaratively. They should be deployable via ArgoCD and allowing users to use optional encryption layer for sensitive properties/objects that may be considered a PII risk (see [`ADR 0008`](./)). Additionally the resulting layout should be flexible enough that migrating a project between clusters and/or extending the project to a multiple clusters is simple enough.
```

Comment:
Rephrase: 
> They should be deployable via ArgoCD and allowing users to use optional encryption layer for sensitive 

to: 

> They should be deployable via ArgoCD and allow users to use an optional encryption layer for sensitive

Reply:
Nit: I'd also remove `,` from `take in the previous ADRs, makes us face additional challenges`

Reply:
`extending the project to a multiple clusters` -> `extending the project to multiple clusters`

Reply:
Thank you, both! :slightly_smiling_face: